.env.example

```
@@ -42,4 +42,10 @@ BASE_URL=http://localhost:8080
 OPENAI_API_KEY=your_openai_api_key
 
 #10 AES
-AES_SECRET=your_aes_secret_key
\ No newline at end of file
+AES_SECRET=your_aes_secret_key
+
+#쿠키 설정 - 로컬에선 비워두면 기본값(false,빈값,Lax)
+#운영 배포는 배포 환경에 맞게 깃허브 secrets 로 관리중(노션 참조)
+COOKIE_SECURE=
+COOKIE_DOMAIN=
+COOKIE_SAME_SITE
```

dotenv-linter (4.0.0)

[warning] [KeyWithoutValue] The COOKIE_SAME_SITE key should be with a value or have an equal sign

(KeyWithoutValue)